pom.xml

```diff
@@ -109,7 +109,7 @@
     <fastutil.version>6.6.1</fastutil.version>
     <geronimo-jms.version>1.1.1</geronimo-jms.version>
     <guava.version>13.0.1</guava.version>
-    <hadoop.version>2.3.0</hadoop.version>
+    <hadoop.version>2.6.0</hadoop.version>
```

Comment:
Is the hadoop version bump needed to support spark 2?

Reply:
It's what in the main repo. Change here is to be consistent with cdapio. I believe we were holding to 2.3 due to spark 1. And AFAIR Spark 3 does not work with 2.3, so 2.6 was a good common denominator

Reply:
Does it work? I believe there would be bunch of unit-test failure? the CDAP core itself has some dependencies on the particular older Hadoop version

Reply:
Yep, it works. This change actually should fix last few failing tests in BUT due to version discrepancy between CDAP and this repo. As for Dataproc, they are already at 2.9 for 1.3